lineman/app/components/thread_page/proposal_pie_chart/proposal_pie_chart.scss

```diff
@@ -7,40 +7,28 @@
   color: $grey-on-white;
   padding-top: 16px;
   padding-left: 20px;
-  td {
-    padding-bottom: 10px;
-  }
+  table { min-width: 110px; }
+  td { padding-bottom: 10px; }
 }
 
 .proposal-pie-chart__label{
-  padding-left: 10px;
-}
-
-.proposal-pie-chart__icon{
-  width: 18px;
-  height: 18px;
-  border-width: 1px;
-  border-style: solid;
-  @include roundedCorners;
+  border-width: 2px;
+  border-bottom-style: solid;
 }
 
-.proposal-pie-chart__icon--yes{
-  background-color: $agree-color;
+.proposal-pie-chart__label--yes{
   border-color: $agree-border-color;
 }
 
-.proposal-pie-chart__icon--abstain{
-  background-color: $abstain-color;
+.proposal-pie-chart__label--abstain{
   border-color: $abstain-border-color;
 }
 
-.proposal-pie-chart__icon--no{
-  background-color: $disagree-color;
+.proposal-pie-chart__label--no{
   border-color: $disagree-border-color;
 }
 
-.proposal-pie-chart__icon--block{
-  background-color: $block-color;
+.proposal-pie-chart__label--block{
   border-color: $block-border-color;
```

Comment:
I'd really like to use the actual colors of the pie chart here, instead of their darkened counterparts... it's looks confusing to me because 'disagree' looks the same as 'block', and 'block' looks maroonish instead of red.